src/scripts/main.js

```diff
@@ -1 +1,91 @@
 'use strict';
+
+let isClickedLeft = false;
+let isClickedRight = false;
+let timer;
+const notification = document.createElement('div');
+
+notification.setAttribute('data-qa', 'notification');
+
+function resetTimer() {
+  clearTimeout(timer);
+
+  timer = setTimeout(() => {
+    if (!isClickedLeft) {
+      firstPromise();
+    }
+  }, 3000);
+}
+
+document.addEventListener('mousedown', (e) => {
+  if (e.button === 0) {
+    isClickedLeft = true;
+    firstPromise();
+    resetTimer();
+  }
+
+  if (e.button === 2) {
+    isClickedRight = true;
+  }
+
+  if (isClickedLeft !== isClickedRight) {
+    secondPromise();
+  }
+
+  if (isClickedLeft && isClickedRight) {
+    thirdPromise();
+  }
+});
+
+document.addEventListener('contextmenu', (e) => {
+  e.preventDefault();
+});
+
+async function firstPromise() {
```

Comment:
The use of the `async` keyword here is unnecessary since you're not using `await` inside the function. This could lead to confusion about how the function is supposed to be used.